<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transport_workspace</class>
 <widget class="QMainWindow" name="transport_workspace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1281</width>
      <height>1011</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>611</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Showcard Gothic&quot;;</string>
      </property>
      <property name="title">
       <string>Formulaire</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Showcard Gothic&quot;;</string>
       </property>
       <property name="text">
        <string>Matricule</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>110</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Showcard Gothic&quot;;</string>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>180</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Showcard Gothic&quot;;</string>
       </property>
       <property name="text">
        <string>Nom de la campagnie</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>260</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Showcard Gothic&quot;;</string>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>340</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Showcard Gothic&quot;;</string>
       </property>
       <property name="text">
        <string>Nombre de place</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_Mat">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>70</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 12px;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_type">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>130</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 12px;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nomcompagnie">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>210</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 12px;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prix">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>280</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 12px;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nbplace">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>370</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 12px;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajout">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>130</y>
         <width>191</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	font: 75 12pt &quot;Bell MT&quot;;
	background-color: rgb(255, 96, 33);
color:rgb(0,0,0);
border-style: solid;
    border-radius: 6;
}
QPushButton:hover
{
	font: 75 12pt &quot;Bell MT&quot;;
	background-color: rgb(255, 164, 52);
color:rgb(0,0,0);
}
QPushButton:pressed
{

	
	font: 75 12pt &quot;Bell MT&quot;;
	background-color: rgb(255, 187, 92);

}</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_email">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>340</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>-70</x>
        <y>-90</y>
        <width>1271</width>
        <height>901</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>C:/Users/21692/Desktop/integration norchene/transport tebe3 nourchene/airplane.webp</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_stat">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>490</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	font: 75 12pt &quot;Bell MT&quot;;
	background-color: rgb(255, 96, 33);
color:rgb(0,0,0);
border-style: solid;
    border-radius: 6;
}
QPushButton:hover
{
	font: 75 12pt &quot;Bell MT&quot;;
	background-color: rgb(255, 164, 52);
color:rgb(0,0,0);
}
QPushButton:pressed
{

	
	font: 75 12pt &quot;Bell MT&quot;;
	background-color: rgb(255, 187, 92);

}</string>
      </property>
      <property name="text">
       <string>Afficher les statistiques</string>
      </property>
     </widget>
     <widget class="QPushButton" name="search_2">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>780</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	font: 75 12pt &quot;Bell MT&quot;;
	background-color: rgb(255, 96, 33);
color:rgb(0,0,0);
border-style: solid;
    border-radius: 6;
}
QPushButton:hover
{
	font: 75 12pt &quot;Bell MT&quot;;
	background-color: rgb(255, 164, 52);
color:rgb(0,0,0);
}
QPushButton:pressed
{

	
	font: 75 12pt &quot;Bell MT&quot;;
	background-color: rgb(255, 187, 92);

}</string>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <zorder>label_13</zorder>
     <zorder>groupBox_2</zorder>
     <zorder>pb_stat</zorder>
     <zorder>search_2</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Afficher</string>
     </attribute>
     <widget class="QTableView" name="Tab_transport">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>521</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>-130</x>
        <y>-70</y>
        <width>1401</width>
        <height>811</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>C:/Users/21692/Desktop/integration norchene/transport tebe3 nourchene/airplane.webp</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_pdf">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>40</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	font: 75 12pt &quot;Bell MT&quot;;
	background-color: rgb(255, 96, 33);
color:rgb(0,0,0);
border-style: solid;
    border-radius: 6;
}
QPushButton:hover
{
	font: 75 12pt &quot;Bell MT&quot;;
	background-color: rgb(255, 164, 52);
color:rgb(0,0,0);
}
QPushButton:pressed
{

	
	font: 75 12pt &quot;Bell MT&quot;;
	background-color: rgb(255, 187, 92);

}</string>
      </property>
      <property name="text">
       <string>Exporter en PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>350</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	font: 75 12pt &quot;Bell MT&quot;;
	background-color: rgb(255, 96, 33);
color:rgb(0,0,0);
border-style: solid;
    border-radius: 6;
}
QPushButton:hover
{
	font: 75 12pt &quot;Bell MT&quot;;
	background-color: rgb(255, 164, 52);
color:rgb(0,0,0);
}
QPushButton:pressed
{

	
	font: 75 12pt &quot;Bell MT&quot;;
	background-color: rgb(255, 187, 92);

}</string>
      </property>
      <property name="text">
       <string>Inserer une image</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_chercher">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>350</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 12px;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>140</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_trier">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>190</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>trier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>240</y>
        <width>171</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_11</zorder>
     <zorder>Tab_transport</zorder>
     <zorder>pb_pdf</zorder>
     <zorder>pushButton_4</zorder>
     <zorder>le_chercher</zorder>
     <zorder>comboBox</zorder>
     <zorder>pb_trier</zorder>
     <zorder>label_18</zorder>
    </widget>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>Supprimer</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>511</width>
        <height>261</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Showcard Gothic&quot;;</string>
      </property>
      <property name="title">
       <string>Suppression</string>
      </property>
      <widget class="QLineEdit" name="le_Mat_supp">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>511</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 12px;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="le_Mat_2">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>50</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Showcard Gothic&quot;;</string>
       </property>
       <property name="text">
        <string>Matricule</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_supprimer">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>180</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	font: 75 12pt &quot;Bell MT&quot;;
	background-color: rgb(255, 96, 33);
color:rgb(0,0,0);
border-style: solid;
    border-radius: 6;
}
QPushButton:hover
{
	font: 75 12pt &quot;Bell MT&quot;;
	background-color: rgb(255, 164, 52);
color:rgb(0,0,0);
}
QPushButton:pressed
{

	
	font: 75 12pt &quot;Bell MT&quot;;
	background-color: rgb(255, 187, 92);

}</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>-30</y>
        <width>1271</width>
        <height>781</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>C:/Users/21692/Desktop/integration norchene/transport tebe3 nourchene/airplane.webp</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>label_12</zorder>
     <zorder>groupBox</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Modifier</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>601</width>
        <height>431</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Showcard Gothic&quot;;</string>
      </property>
      <property name="title">
       <string>Modification</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Matricule</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>110</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>180</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom de la compagnie</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>250</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>320</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nombre de place</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_Mat_3">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 12px;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_type_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>140</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 12px;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nomcompagnie_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>210</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 12px;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prix_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>280</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 12px;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nbplace_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>360</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 12px;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_modifier">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>120</y>
         <width>191</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	font: 75 12pt &quot;Bell MT&quot;;
	background-color: rgb(255, 96, 33);
color:rgb(0,0,0);
border-style: solid;
    border-radius: 6;
}
QPushButton:hover
{
	font: 75 12pt &quot;Bell MT&quot;;
	background-color: rgb(255, 164, 52);
color:rgb(0,0,0);
}
QPushButton:pressed
{

	
	font: 75 12pt &quot;Bell MT&quot;;
	background-color: rgb(255, 187, 92);

}</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>-50</x>
        <y>-110</y>
        <width>1331</width>
        <height>981</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>C:/Users/21692/Desktop/integration norchene/transport tebe3 nourchene/airplane.webp</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>label_14</zorder>
     <zorder>groupBox_3</zorder>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Sortie</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>120</y>
        <width>160</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="ouvrir_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Ouvrir</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>220</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Places disponibles :</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>250</y>
        <width>151</width>
        <height>78</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>90</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Entrée</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>160</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="ouvrir">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Ouvrir</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>0</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>927</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
